lora_pkt_fwd/src/lora_pkt_fwd.c

```diff
@@ -2596,17 +2600,20 @@ void thread_down(void) {
             if (jit_result == JIT_ERROR_OK) {
                 uint8_t tx_lut_idx = 0;
                 i = get_tx_gain_lut_index(txpkt.rf_power, &tx_lut_idx);
-                if ((i < 0) || (txlut.lut[tx_lut_idx].rf_power != txpkt.rf_power)) {
+                if (i < 0) {
+                    jit_result = JIT_ERROR_TX_POWER;
+                    printf("WARNING: No TX power lower or equal to %ddBm is supported, rejecting transmission\n", txpkt.rf_power);
```

Comment:
haven't these values been converted from `ddBm` (really `cBm`) to `dBm` by this point?

Reply:
Hah, of course. Sorry about that

Reply:
can you repoint the submodule to the merged helium/lora_gateway sha?